django-backend/fecfiler/settings/base.py

```diff
@@ -1,19 +1,15 @@
 """
-Django settings for FEC-EFiler project.
+Django settings for the FECFile project.
 
-Django 1.11.
-
-For more information on this file, see
-https://docs.djangoproject.com/en/1.11/topics/settings/
-
-For the full list of settings and their values, see
-https://docs.djangoproject.com/en/1.11/ref/settings/
 """
 
 import os
 import datetime
+import dj_database_url
+
+from .env import env
 from corsheaders.defaults import default_headers
-import logging
+from django.utils.crypto import get_random_string
 
 
 CORS_ALLOWED_ORIGINS = [
```

Comment:
When I access the cloud.gov [front end](https://fecfile-web-app-dev.app.cloud.gov/#/) and login, I'm seeing some CORS errors in the console.  It's an odd thing because some API responses do have the proper `access-control-allow-origin: *
` header and others don't.  For example https://fecfile-web-api-dev.app.cloud.gov/api/v1/user/login/authenticate does have the header and so login works fine.     

The ones that don't have the CORS header generate errors in the browser dev console:
`Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remote resource at https://fecfile-web-api-dev.app.cloud.gov/api/v1/core/recent_saved_reports. (Reason: CORS header ‘Access-Control-Allow-Origin’ missing). Status code: 502.`
It seems strange that some are getting the header and others don't.  If this is an simple fix, it is in scope to do it in this ticket and it makes sense to get it right - but if it is a bigger problem or a hard fix, I would support rolling it into a new issue.

Reply:
Thanks for flagging this, it looks like the `gunicorn` workers were rebooting due to high CPU, resulting in 502/503 errors without CORS headers. I redeployed with more memory, which looks like that took care of it. Can you take a look and see if it's working properly now?

Reply:
I just tested at https://fecfile-web-app-dev.app.cloud.gov/ and all my CORS errors are gone now.  So this looks like a fix to me.